backend/src/Designer/Controllers/ResourceAdminController.cs

```diff
@@ -138,6 +139,69 @@ public async Task<ActionResult> RemoveResourceAccessList(string env, string org,
             return new StatusCodeResult(((int)statusCode));
         }
 
+        [HttpGet]
+        [Route("designer/api/{org}/resources/allaccesslists")]
+        public async Task<ActionResult> GetAllAccessLists(string org)
+        {
+            // sjekk at man er medlem av minst et Resources-Publish-XXXX team for å kunne kalle dette endepunktet
```

Comment:
Should this comment be there? If yes, maybe it should be in english as the rest of the code? 😄